features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/Resources.js

```diff
@@ -120,6 +141,19 @@ class Resources extends React.Component{
                     break;
                 case "POST" :
                     pathValue["POST"] = defaultPost;
+                    break;
+                case "PUT" :
```

Comment:
IMHO , we can replace the switch condition with `pathValue[method] = defaultPost;`